### DISCUSSION TITLE
Does perplexity API currently support reading images?

### DISCUSSION THREAD

**Discussion by yshlin** [perplexityai/api-discussion#20] [Q&A] [answered]
I'm trying to use it to recognize diagrams, but can't seem to find any documentation.
Putting image URLs or base64 image in the payload also won't work

**Comment by shubhang98** [comment, marked as answer]
We do not support image uploads in the API right now.

**Reply by ijazul-haq** [reply]
I should have read this comment before payment 😭

**Comment by mkhanplative** [comment]
Is this planned anytime soon?
